synapse/telepath.py

```diff
@@ -426,8 +445,13 @@ def sockfini():
         # add the sock to the multiplexor
         self._tele_plex.addPlexSock(sock)
 
-        # let client code do stuff on reconnect
         self._tele_sock = sock
+
```

Comment:
The ``tele:sock:init`` message is already used by the proxy to do post-socket init actions (re-registering on handlers). Do we want to fire reminders in response to that message? Do we want reminders fired before or after on handlers are reregistered?